client/sites/components/site-preview-pane/dotcom-preview-pane.tsx

```diff
@@ -97,7 +98,7 @@ const DotcomPreviewPane = ( {
 						<HostingFeaturesIcon />
 					</span>
 				),
-				enabled: isSimpleSite || isPlanExpired,
+				enabled: areHostingFeaturesSupported( site ),
```

Comment:
I believe this should be 

```suggestion
				enabled: ( isSimpleSite || isPlanExpired ) && ! config.isEnabled( 'untangling/hosting-menu' ),
```

because we want to still show this when the feature flag is not enabled.

Reply:
fixed in 6c57e7fc9b7aa4809883f5b10d01e15ce006ab26